src/platform/packages/shared/kbn-unified-data-table/src/components/custom_toolbar/render_custom_toolbar.tsx

```diff
@@ -90,18 +91,28 @@ export const internalRenderCustomToolbar = (
         <EuiFlexItem grow={false}>
           <EuiFlexGroup responsive={false} gutterSize="s" alignItems="center">
             {Boolean(leftSide) && buttons}
-            {(keyboardShortcutsControl || displayControl || fullScreenControl) && (
+            {Boolean(
```

Comment:
super nit: all the boolean calls for each of these can probably be done once in the body of the component and not the render,